### PR TITLE
Feature/update cappuccino

### PR DESCRIPTION
**Pull request by MizukiSonoko**
Sorry I'm late.
Cappuccino uses Asio standalone.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -38,10 +38,12 @@ test:
     # TESTS:
     # first, run ctest inside a container
     - docker run -p 1204:1204 hyperledger/iroha-docker /test.sh
+
+    - docker run -d -p 1204:1204 hyperledger/iroha-docker /run.sh && sleep 5 && curl http://127.0.0.1:1204
```
Comment (Warchant) [inline circle.yml]: Look closely, you run two containers on the same port.

Should be:
```
- docker run -d -p 1204:1204 hyperledger/iroha-docker /run.sh
- sleep 5
- curl http://127.0.0.1:1204
- curl -X POST http://127.0.0.1:1204/account/register -d '{"publicKey":"WdvM/DPabapmtA7ISbTYPywbHxk8gWu2221LzmcmAgw=","alias":"yonezu","timestamp":1482053586}'
```

File: core/model/commands/remove.hpp [hunk 2]
```diff
@@ -14,22 +14,36 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+
+        template<typename... Args>
+        constexpr Remove(
+            Args&&... args
+        ):
+            T(std::forward<Args>(args)...)
+        {}
 
-    template <>
-    void Add<object::Asset>::execution() {
```
Comment (takemiyamakoto) [inline core/model/commands/remove.hpp]: Codacy Issue found: [Class 'Remove' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/pullRequest?prid=485945)

File: core/model/commands/update.hpp [hunk 3]
```diff
@@ -14,43 +14,34 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+        constexpr auto getCommandName() const {
             return "Update";
         }
-
-        void execution(){
-
-        }
-
     };
-
```
Comment (takemiyamakoto) [inline core/model/commands/update.hpp]: Codacy Issue found: [Class 'Update' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/pullRequest?prid=485945)

**Comment by takemiyamakoto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 20
           
```


See the complete overview on [Codacy](https://www.codacy.com/app/soramitsu/iroha/pullRequest?prid=485945&bid=4095525)

File: circle.yml (hunk 1 above)
Comment (MizukiSonoko) [inline circle.yml]: Ok thanks, I will  remove.
And currently, this test is failed, I'm trying to fix it. 🏃 
```
curl -X POST http://127.0.0.1:1204/account/register -d '{"publicKey":"WdvM/DPabapmtA7ISbTYPywbHxk8gWu2221LzmcmAgw=","alias":"yonezu","timestamp":1482053586}' 
```

**Comment by Warchant** [comment]
I'll resolve conflcts

**Comment by Warchant** [comment]
Now it can be merged.

File: docker/dev/Dockerfile [hunk 4]
```diff
@@ -69,7 +69,7 @@ RUN apt-get update && apt-get -y upgrade; \
     cd /tmp; \
     git clone --recursive -b $(curl -L http://grpc.io/release) https://github.com/grpc/grpc; \
     cd grpc; \
-    make && make install
+    make -j4 && make install
```
Comment (l4l) [inline docker/dev/Dockerfile]: Why 4?

**Pull request merged**